packages/backend/src/accounting/psql/service.ts

```diff
@@ -13,16 +13,27 @@ import {
   Withdrawal
 } from '../service'
 import {
+  LedgerAccount,
   LedgerAccountType,
   mapLiquidityAccountTypeToLedgerAccountType
 } from './ledger-account/model'
 import { LedgerAccountService } from './ledger-account/service'
+import { LedgerTransfer, LedgerTransferState } from './ledger-transfer/model'
+import { LedgerTransferService } from './ledger-transfer/service'
 
 export interface ServiceDependencies extends BaseService {
   ledgerAccountService: LedgerAccountService
+  ledgerTransferService: LedgerTransferService
```

Comment:
Would definitely appreciate some thoughts & opinions on service structure. Maybe instead, I keep everything (account lookups & transfer lookups) under this single AccountingService and then pull everything out later as necessary?

Reply:
Maybe a `BalanceService` or so? Or maybe just mimick the Tigerbeetle service setup directly?

Reply:
I think we should either have this service be an integration service that just pools other services and not have its own business logic or this service include all business logic. I don't like that it's mixed right now.

Reply:
@sabineschaller I was thinking about this too, I think we can just do sort of like the tigerbeetle services, where there are separate functions in separate files that take in `deps: ServiceDependencies`. Maybe something like `balances.ts` and `transfers.ts` `accounts.ts`

Reply:
I'd like to just merge this in and do a new PR for splitting the services out, if that makes sense @sabineschaller ?